MIDAS/src/hardware/Continuity.cpp

```diff
@@ -1,17 +1,53 @@
 #include "sensors.h"
 #include "ads7138-q1.h"
 #include <hal.h>
+#include <Wire.h>
 
 #define PYRO_VOLTAGE_DIVIDER (5.0 / (5.0 + 20.0))       //voltage divider for pyro batt voltage, check hardware schematic
 #define CONT_VOLTAGE_DIVIDER (5.0 / (5.0 + 20.0))       //voltage divider for continuity voltage, check hardware schematic
 
+// Reads the given register from the pyro power monitor
+int read_reg(int reg, int bytes) {
```

Comment:
can you call this something specific to continuity so it doesn't clash with other names